src/multivariate/dirichlet.jl

```diff
@@ -381,26 +371,32 @@ function fit_dirichlet!(elogp::Vector{Float64}, α::Vector{Float64};
         converged = gnorm < tol
     end
 
+    if !converged
+        throw(ErrorException("no convergence after $maxiter (maxiter) iterations"))
```

Comment:
```suggestion
        throw(ErrorException("No convergence after $maxiter (maxiter) iterations."))
```

Reply:
In Julia Base, error messages typically don't start will a capital and the don't end with a period. It looks like we are pretty inconsistent here and I'll file an issues about it.

Reply:
I just saw the style of the one below

Reply:
Fair point. Also one like that above it. Within function error message consistency seems like a pretty low requirement.